libimage/copier.go

```diff
@@ -206,13 +206,17 @@ func getDockerAuthConfig(name, passwd, creds, idToken string) (*types.DockerAuth
 	}
 }
 
+// NewCopier is a simple, exported wrapper for newCopier
```

Comment:
“exported wrapper” is not really a useful comment for the external consumer of this type.

Maybe reuse the comment from the existing `newCopier`? That doesn’t say much, but _at least_ it says to call `.Close` and show the context and options interact.